efel/cppcore/LibV1.cpp

```diff
@@ -271,7 +272,7 @@ int LibV1::firing_rate(mapStr2intVec& IntFeatureData,
     return nSize;
   else {
     vector<double> stimStart, stimEnd, peakVTime, firing_rate;
-    double lastAPTime;
+    double lastAPTime = 0.;
```

Comment:
Good question, it was uninitialized before, so that's why I set it, b/c it's used at the end of the function, iirc.  A better value should be picked in the case that line ~293 doesn't get hit